### PR TITLE
[tests] Fix monorepo tests

### PR DESCRIPTION
**Pull request by chloetedder**

### CODE REVIEW

**Review by cb1kenobi — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "build": "node utils/gen.js && turbo run build",
     "vercel-build": "pnpm build && pnpm run pack && cd api && node -r ts-eager/register ./_lib/script/build.ts",
     "pre-commit": "lint-staged",
-    "test": "jest --rootDir=\"test\" --testPathPattern=\"\\.test.js\"",
+    "test": "jest --rootDir=\"test\" --testPathPattern=\"\\.test.ts\"",
```
Comment (styfle) [inline package.json]: I don't think this is correct

**Review by styfle — CHANGES_REQUESTED** [review]
Did you intend to change the root `package.json`?

File: packages/cli/test/fixtures/unit/commands/build/monorepo-detection/nx-package-config/packages/app-2/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "app-2",
   "version": "0.0.1",
-  "main": "dist/index.js",
+  "main": "script.js",
```
Comment (styfle) [inline packages/cli/test/fixtures/unit/commands/build/monorepo-detection/nx-package-config/packages/app-2/package.json]: What's the purpose of this change?

I would have expected `main` be the dist output instead of the source file.

Comment (cb1kenobi) [reply packages/cli/test/fixtures/unit/commands/build/monorepo-detection/nx-package-config/packages/app-2/package.json]: ~There is no `dist/index.js` in this monorepo test fixture.~

File: package.json (hunk 1 above)
Comment (cb1kenobi) [reply package.json]: Agree w/ @styfle. There are no `.ts` files in the root `test` directory.

File: packages/cli/test/fixtures/unit/commands/build/monorepo-detection/nx-package-config/packages/app-2/package.json (hunk 2 above)
Comment (styfle) [reply packages/cli/test/fixtures/unit/commands/build/monorepo-detection/nx-package-config/packages/app-2/package.json]: @cb1kenobi If you look at the build script, it emits a `dist/index.js` which is typically how npm packages work: `main` points to the compiled output file

https://github.com/vercel/vercel/blob/main/packages/cli/test/fixtures/unit/commands/build/monorepo-detection/nx-package-config/packages/app-2/script.js

File: packages/cli/test/fixtures/unit/commands/build/monorepo-detection/turbo-package-config/packages/app-2/package.json [hunk 3]
```diff
@@ -1,10 +1,6 @@
 {
   "name": "app-2",
   "version": "0.0.1",
-  "main": "dist/index.js",
-  "files": [
-    "dist"
-  ],
   "scripts": {
```
Comment (styfle) [inline packages/cli/test/fixtures/unit/commands/build/monorepo-detection/turbo-package-config/packages/app-2/package.json:4]: Can you explain why this would affect the test? Do we read “main” or “files” at all?
